src/app/home/home.component.html

```diff
@@ -1,18 +1,25 @@
-<div class="dashboard-content">
-  <div class="row">
-    <div class="col-md-2" *ngFor="let info of homedisplayArray">
-      <mat-card class="product">
-        <mat-card-content>
-          <div class="roundimg">
-            <mat-icon class="mat-icon-custom" matTooltipPosition="above" matTooltip="{{info.title}}"><a
-                routerLink="{{info.routerlink}}">{{info.icon}}</a></mat-icon>
-          </div>
-          <div class="text-center">
-            <h5>{{info.title}}</h5>
-            <span (click)="singleuser()">{{info.data}}</span>
-          </div>
-        </mat-card-content>
-      </mat-card>
-    </div>
+<div class="pageName">
+  Admin Console
+</div>
+  <div *ngIf = "response" class="page-loading-shade">
```

Comment:
create a component for loading and reuse

Reply:
Created and using the same